requirements.txt

```diff
@@ -1,2 +1,3 @@
 Jinja2==2.10
 jinja2-cli==0.6.0
+URLObject==2.4.3
```

Comment:
Not sure if it is worth pulling in a dependency that bundles six and looks unmaintained for the tiny bit of functionality.